KubernetesConfigWithSecretsTest is failing the build

Various CI jobs and PRs have been failing with something like:

```
2020-09-25T23:32:02.6892619Z [ERROR] io.quarkus.it.kubernetes.KubernetesConfigWithSecretsTest.assertGeneratedResources  Time elapsed: 0.046 s  <<< FAILURE!
2020-09-25T23:32:02.6897491Z java.lang.AssertionError: 
2020-09-25T23:32:02.6898863Z 
2020-09-25T23:32:02.6901677Z Expecting any element of:

....


```

--- a/integration-tests/kubernetes/quarkus-standard-way/src/test/java/io/quarkus/it/kubernetes/KubernetesConfigWithSecretsTest.java
+++ b/integration-tests/kubernetes/quarkus-standard-way/src/test/java/io/quarkus/it/kubernetes/KubernetesConfigWithSecretsTest.java
@@ -62,39 +62,37 @@ public class KubernetesConfigWithSecretsTest {
             });
         });
 
-        assertThat(kubernetesList).filteredOn(h -> "RoleBinding".equals(h.getKind())).hasSize(2);
+        assertThat(kubernetesList).filteredOn(h -> "RoleBinding".equals(h.getKind())).hasSize(2)
+                .anySatisfy(res -> {
+                    assertThat(res).isInstanceOfSatisfying(RoleBinding.class, roleBinding -> {
+                        assertThat(roleBinding.getMetadata()).satisfies(m -> {
+                            assertThat(m.getName()).isEqualTo("kubernetes-config-with-secrets-view-secrets");
+                        });
 
-        assertThat(kubernetesList).anySatisfy(res -> {
-            assertThat(res).isInstanceOfSatisfying(RoleBinding.class, roleBinding -> {
-                assertThat(roleBinding.getMetadata()).satisfies(m -> {
-                    assertThat(m.getName()).isEqualTo("kubernetes-config-with-secrets:view-secrets");
+                        assertThat(roleBinding.getRoleRef().getKind()).isEqualTo("Role");
+                        assertThat(roleBinding.getRoleRef().getName()).isEqualTo("view-secrets");
+
+                        assertThat(roleBinding.getSubjects()).singleElement().satisfies(subject -> {
+                            assertThat(subject.getKind()).isEqualTo("ServiceAccount");
+                            assertThat(subject.getName()).isEqualTo("kubernetes-config-with-secrets");
+                        });
+                    });
+                })
+                .anySatisfy(res -> {
+                    assertThat(res).isInstanceOfSatisfying(RoleBinding.class, roleBinding -> {
+                        assertThat(roleBinding.getMetadata()).satisfies(m -> {
+                            assertThat(m.getName()).isEqualTo("kubernetes-config-with-secrets-view");
+                        });
+
+                        assertThat(roleBinding.getRoleRef().getKind()).isEqualTo("ClusterRole");
+                        assertThat(roleBinding.getRoleRef().getName()).isEqualTo("view");
+
+                        assertThat(roleBinding.getSubjects()).singleElement().satisfies(subject -> {
+                            assertThat(subject.getKind()).isEqualTo("ServiceAccount");
+                            assertThat(subject.getName()).isEqualTo("kubernetes-config-with-secrets");
+                        });
+                    });
                 });
-
-                assertThat(roleBinding.getRoleRef().getKind()).isEqualTo("Role");
-                assertThat(roleBinding.getRoleRef().getName()).isEqualTo("view-secrets");
-
-                assertThat(roleBinding.getSubjects()).singleElement().satisfies(subject -> {
-                    assertThat(subject.getKind()).isEqualTo("ServiceAccount");
-                    assertThat(subject.getName()).isEqualTo("kubernetes-config-with-secrets");
-                });
-            });
-        });
-
-        assertThat(kubernetesList).anySatisfy(res -> {
-            assertThat(res).isInstanceOfSatisfying(RoleBinding.class, roleBinding -> {
-                assertThat(roleBinding.getMetadata()).satisfies(m -> {
-                    assertThat(m.getName()).isEqualTo("kubernetes-config-with-secrets:view");
-                });
-
-                assertThat(roleBinding.getRoleRef().getKind()).isEqualTo("ClusterRole");
-                assertThat(roleBinding.getRoleRef().getName()).isEqualTo("view");
-
-                assertThat(roleBinding.getSubjects()).singleElement().satisfies(subject -> {
-                    assertThat(subject.getKind()).isEqualTo("ServiceAccount");
-                    assertThat(subject.getName()).isEqualTo("kubernetes-config-with-secrets");
-                });
-            });
-        });
     }
 
 }

--- a/integration-tests/kubernetes/quarkus-standard-way/src/test/java/io/quarkus/it/kubernetes/WithKubernetesClientTest.java
+++ b/integration-tests/kubernetes/quarkus-standard-way/src/test/java/io/quarkus/it/kubernetes/WithKubernetesClientTest.java
@@ -67,7 +67,7 @@ public class WithKubernetesClientTest {
         });
 
         assertThat(kubernetesList).filteredOn(h -> "RoleBinding".equals(h.getKind())).singleElement().satisfies(h -> {
-            assertThat(h.getMetadata().getName()).isEqualTo("kubernetes-with-client:view");
+            assertThat(h.getMetadata().getName()).isEqualTo("kubernetes-with-client-view");
         });
     }
 }

